task_templates/estimators/python_multiclass_classification/model-metadata.yaml

```diff
@@ -1,25 +1,36 @@
+# model-metadata.yaml allows to specify task input/output requirements to help validate a blueprint where it's used.
+# It's also valuable if you are using `drum push`
 # see MODEL-METADATA.md and VALIDATION-SCHEMA.md for full explanations of fields
 name: python_multiclass_classification
-type: training
-targetType: multiclass
-# modelID: optional, used with drum push to create a new version of an existing model
-# environmentID: optional, used with drum push to select environment
-#trainingModel:
-#  trainOnProject: optional project on which to train
-validation:
-  input: ../../../tests/testdata/skyserver_sql2_27_2018_6_51_39_pm.csv
-  targetName: class
+type: training # training (for custom tasks) or inference (for custom inference models)
+environmentID: 5e8c889607389fe0f466c72d # optional, only required when using "drum push" to select environment
```

Comment:
This needs to match the most recent environmentID on prod

Reply:
and make sure the env matches the contents of the custom task template, ie pytorch env for pytorch model, sklearn env for sklearn models, etc

Reply:
@jmbannon ahh, good catch. They matched the environments for 1.5.14 but not the latest drum version. 

But what do you mean "it might be easier excluding them?"

Reply:
Ohh, do you mean just getting rid of environment ID because it's optional except for drum push?

Reply:
@jmbannon I confirmed that every environmentID matches the appropriate environment on Prod. I used the environment ID at the location below for each different environment:

![image](https://user-images.githubusercontent.com/508707/135108395-8aeadd23-92db-47c2-ad8c-88e42b10e529.png)

Reply:
oh nice, i was comparing this one to the sklearn one